pageserver/src/repository.rs

```diff
@@ -163,6 +163,12 @@ pub trait Timeline: Send + Sync {
     /// NOTE: This has nothing to do with checkpoint in PostgreSQL. We don't
     /// know anything about them here in the repository.
     fn checkpoint(&self) -> Result<()>;
+
+    ///
+    /// Wait until the repository receives given LSN. Right now it is only used
```

Comment:
Let's mention that most functions do this implicitly, and list the functions that don't. (I think we should just make all of them call this implicitly)